### ISSUE TITLE
Help With Crafting. It won't work.

### ISSUE THREAD

**Issue by PhilipDeneva** [elBukkit/MagicPlugin#372]
I'm new to this whole running a server thing and know nothing about the config files or even what may be needed. I've worked out a lot on my own, but the crafting for this plugin isn't working for some reason.
I need to know if there are any other requirements I may be missing that would not allow the crafting of items.

Thank you for you time,
Philip, the friendly admin.

**Comment by NathanWolf** [comment]
No, if you are using the default configs then crafting of most items is enabled by default. No permissions are required.

You are trying a nether star on top of a blaze rod, and nothing comes out?
Is the plugin working, otherwise?
What server version and plugin version do you have?

**Comment by NathanWolf** [comment]
Other than yaml annoyances the syntax is pretty simple:
```
broomstick:
  enabled: false
```

So the name of the recipe, a colon, new line, at least one space (not a tab) and the phrase `enabled: false` - make sure there's a space between the colon and `false`.

**Comment by PhilipDeneva** [comment]
![image](https://user-images.githubusercontent.com/38973613/39642590-72cfd146-4f8f-11e8-8665-adf5f0ce22f3.png)

![image](https://user-images.githubusercontent.com/38973613/39642640-9949c98a-4f8f-11e8-97cd-ddb76fa10471.png)

I know I did the syntax exactly as you said, It broke the plugin again. The textures aren't showing up and I'm not even able to use the items.

Thank you for your trouble. 
Philip, a desperate admin

**Comment by NathanWolf** [comment]
zombie_leather needs a colon after it :)

If you want, you can paste your configs in here:

http://yaml-online-parser.appspot.com

So you know if they're broken before deploying on the server.

**Comment by PhilipDeneva** [comment]
Oh, I feel dumb now. Thank you so much. Thank you for your patience. Just one more thing. For the Rabbit Boots, do I need an underscore or do I leave the space out in the config?

**Comment by PhilipDeneva** [comment]
Thank you so much.
This will be so much help, you don't even know how much I've been stressing over this server. Honestly, it's become like my baby. It gets so much attention. Thank you for being willing to provide any needed information. If I ever need help again, I'll drop a question.

With the utmost gratitude,
Philip, the revealed admin.

**Comment by PhilipDeneva** [comment]
3 hours and I already have another question that I can't for the life of me figure out. I even went the elmakers server to try and find an answer. I have no idea how the classes work. I know they are needed to advance a lot of spells beyond a point but I cannot find anything on the matter.

**Comment by NathanWolf** [comment]
This is a good reference for classes: https://github.com/elBukkit/MagicPlugin/wiki/Classes

And this for paths: https://github.com/elBukkit/MagicPlugin/wiki/Progression

Paths are basically like "levels", they define how a player levels up. Generally they contain a list of spells, those are the spells players can get at that level. When they get all of those spells, they level up to the next level as defined by the path.

Classes are new and a little more complicated, but essentially they are a way to compartmentalize player data. It used to be that there was no player progression in this plugin, everything was stored on the wand item (current path, spells, mana, etc). As of 7.0 progression is stored on the player, but specifically on a player class.

So the classes are really just there to hold data like the spells a player has unlocked, their current mana level.

Different classes can be made to separate this data, for instance a player may be a magic wand user and a bender, you'd want their bending progression to be separate from their wand progression.

Let me know if you have more questions!
